easystacks/software.eessi.io/2023.06/icelake_cclake/072-eb-5.0.0.yml

```diff
@@ -0,0 +1,17 @@
+# 072-eb-5.0.0.yml: total build duration = ?? minutes
+easyconfigs:
+  - /cvmfs/software.eessi.io/versions/2023.06/software/linux/x86_64/intel/haswell/software/MUSCLE/5.1.0-GCCcore-12.3.0/easybuild/MUSCLE-5.1.0-GCCcore-12.3.0.eb:
+      options:
+        include-easyblocks: /cvmfs/software.eessi.io/versions/2023.06/software/linux/x86_64/intel/haswell/software/MUSCLE/5.1.0-GCCcore-12.3.0/easybuild/reprod/easyblocks/*.py
+  - /cvmfs/software.eessi.io/versions/2023.06/software/linux/x86_64/intel/haswell/software/Gblocks/0.91b/easybuild/Gblocks-0.91b.eb:
+      options:
+        include-easyblocks: /cvmfs/software.eessi.io/versions/2023.06/software/linux/x86_64/intel/haswell/software/Gblocks/0.91b/easybuild/reprod/easyblocks/*.py
+  - /cvmfs/software.eessi.io/versions/2023.06/software/linux/x86_64/intel/haswell/software/RAxML/8.2.13-gompi-2023a-avx2/easybuild/RAxML-8.2.13-gompi-2023a-avx2.eb:
```

Comment:
The problem here is that this originally only appears as a dep and the version installed is arch dependent, including it here directly causes problems on Arm.

I think if you just remove it from here, all will be right in the world

Reply:
Right, removing it!